### PR TITLE
Meal details screen

### PR DESCRIPTION
**Pull request by gilrPixellot**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
 import React, { useState } from "react";
-import { StyleSheet, View, Text } from "react-native";
+import { StyleSheet, View, Text, I18nManager } from "react-native";
 import WelcomeScreen from "./screens/WelcomeScreen";
 
+try {
+  I18nManager.forceRTL(true);
+} catch (e) {
+  console.log(e);
```
Comment (efi411) [inline App.js:8]: I thing console.error(e) is better in this case.

File: components/LineText.js [hunk 2]
```diff
@@ -7,21 +7,35 @@ import React from "react";
...
+  if (lineColor) {
+    addedLineProps["borderColor"] = lineColor;
+  }
   return (
-    <View>
-      <Text style={{ ...styles.text, ...props.style }}>{props.children}</Text>
-      <View style={styles.lineStyle} />
+    <View style={{ ...styles.comp, ...props.style }}>
+      <Text style={{ ...styles.text, ...textStyle }}>{props.children}</Text>
+      <View style={{ ...styles.lineStyle, ...addedLineProps }} />
```
Comment (efi411) [inline components/LineText.js:23]: Why not add here: ...props.lineStyle and add this style from the outside of the component?
All the "if" statement and the new object "addedLineProps" could be deleted.

File: components/MealDetailCircle.js [hunk 3]
```diff
@@ -4,19 +4,27 @@
  *******************************************/
 
 import React from "react";
-import { StyleSheet, View, Text } from "react-native";
+import { StyleSheet, View, Text, ColorPropType } from "react-native";
```
Comment (efi411) [inline components/MealDetailCircle.js:7]: Where do you use this component? "ColorPropType"

Comment (gilrPixellot) [reply components/MealDetailCircle.js:7]: it happened automatically by mistake

File: components/LineText.js [hunk 4]
```diff
@@ -7,21 +7,31 @@ import React from "react";
...
+    borderColor: lineColor ? lineColor : Color.secondaryColor,
+    margin: textStyle && textStyle["fontSize"] ? textStyle["fontSize"] / 2 : 6,
+  };
+
   return (
-    <View>
-      <Text style={{ ...styles.text, ...props.style }}>{props.children}</Text>
-      <View style={styles.lineStyle} />
+    <View style={{ ...styles.comp, ...props.style }}>
+      <Text style={{ ...styles.text, ...textStyle }}>{props.children}</Text>
```
Comment (efi411) [inline components/LineText.js:21]: Please replace all Text tags with DefaultText tag

File: components/LineText.js [hunk 5]
```diff
@@ -7,21 +7,31 @@ import React from "react";
...
 };
 
 const styles = StyleSheet.create({
   lineStyle: {
-    borderWidth: 3,
-    borderColor: Color.primaryColor,
-    margin: 10
+    width: "80%",
+  },
+  text: {},
```
Comment (efi411) [inline components/LineText.js:31]: What is this property used for?

**Pull request merged**